index.js

```diff
@@ -23,44 +28,44 @@ function healthGain(food) {
     player.inventory.splice(player.inventory.indexOf(food), 1)
     player.currentHealth = 7
     console.log("You restored all your health")
-  } 
+  }
   // if statement for eating a snickers
   else if (player.inventory.includes(food) && food === 'snickers') {
     player.inventory.splice(player.inventory.indexOf(food), 1)
     player.currentHealth = player.currentHealth + 4
     console.log("You gained 5 points of health, but time is still running out!")
-  } 
+  }
   //if statement for eating garlic
   else if (player.inventory.includes(food) && food === 'garlic') {
     player.inventory.splice(player.inventory.indexOf(food), 1)
     player.currentHealth = player.currentHealth + 1
     console.log("You have gained 1 point of health, but at what cost?")
-  } 
+  }
   //if statement for eating candycorn
   else if (player.inventory.includes(food) && food === 'candycorn') {
     player.inventory.splice(player.inventory.indexOf(food), 1)
     player.currentHealth = player.currentHealth + 1
     console.log("You pop in some of the original candycorn, circa 1922. Although your integrity is questioned, you gain 1 point of health")
-  } 
+  }
   //console log for something other then these foods
   else {
     console.log("You cant eat that!")
   }
 }
 
 // function to pick up objects
-function takeStuff(thingIWant) { 
+function takeStuff(thingIWant) {
   let canITakeIt = itemLookUp[thingIWant]
   //seeing if the object even exists in the game
   if (!canITakeIt) {
     console.log('I\'m not sure that exists here. Want to try something else?')
-  } 
+  }
   //if the object does exist and it can be taken.
   else if (canITakeIt.takeable === true && player.location.inv.includes(thingIWant)) {
     player.location.inv.splice(player.location.inv.indexOf(thingIWant), 1)
     player.inventory.push(thingIWant)
     console.log("You have now picked up " + thingIWant)
-  } 
+  }
```

Comment:
This doesn't quite do what you think it does. You're trying to do XYZ but it's actually doing ABC